components/elements/formCheckbox/index.tsx

```diff
@@ -1,7 +1,12 @@
-import { string } from "prop-types";
 import { useSignatureState } from "../../context";
 
-const FormCheckbox = ({ name, label, id, ...rest }) => {
+interface FormCheckboxProps {
+  name: string;
+  label: string;
+  id: string;
+}
+
+const FormCheckbox = ({ name, label, id, ...rest }: FormCheckboxProps) => {
```

Comment:
could be `const FormCheckbox: React.FC<FormCheckboxProps>`

Reply:
As above.